meetup/services/meetup_api_lookup_agent.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from django.http import HttpResponse
+import requests
+import os
+
+class MeetupApiLookupAgent():
+  def __init__(self, link):
+    self.link = link
+
+  def get_response(self, category=''):
+    api_url = "https://api.meetup.com/" + self.get_urlname() + '/' + category
+    url_components = {"key": os.getenv("MEETUP_KEY")}
```

Comment:
Instead of reading this from the environment on every request, a common pattern is to read this once, when you load your Django settings files. Then you can `from django.conf import settings` here and simply access `settings.MEETUP_KEY`. Good job keeping configuration separate from code!